### PR TITLE
Update Mycelium description

### PR DESCRIPTION
**Pull request by saivann**
As suggested in #896, this pull request updates Mycelium's description, and drops any mention of it being open-source.

In the absence of critical feedback, this pull request will be merged on June 17th.

### CODE REVIEW

**Comment by harding** [comment]
@saivann thanks!  Except for the comment to Craig below, this LGTM.

@crwatkins I know you had some concerns with another wallet that listed compatibility with hardware wallets that weren't really supported.  Do you think this needs investigation before being updated?

**Comment by crwatkins** [comment]
@harding I can personally confirm (from previous testing) that Mycelium on Android is compatible with Trezor.  I can further confirm that MultiBit HD with the same Trezor can access the same account as Mycelium with that Trezor.  That's compatible in my book.  However, I don't believe that I would consider Mycelium compatible with MultiBit HD (without a Trezor).   I believe that (without a Trezor) Mycelium uses a BIP44 path of m/44'/0'/0'/c/i while MultiBit HD uses a BIP32 path of m/0'/c/i.  

My suggested fix with the least number of words changed would be "... offers compatibilty with Trezor, Multibit HD with Trezor, and others."  However, that's still not as clear as it could be.  Perhaps @apetersson can confirm this and suggest something that is clearer.

**Comment by harding** [comment]
@crwatkins wow, thanks!  Let's see what @apetersson says.

**Comment by jim618** [comment]
@crwatkins Primarily to cater for the use case of someone losing their Trezor and needing to recover their bitcoins in MultiBit HD with no replacement Trezor we programmed up that the user can create what we call a 'soft' wallet that uses a path of m/44'/0'/0'/c/i.

This is a wallet stored on the desktop computer with all the private keys stored locally and the keys use the Trezor path derivation of m/44'/0'/0'/c/i 

It is available in the restore as an option - see screenshot:

screen shot 2015-06-17 at 09 07 22

Thus more technically minded users _could_ restore their Mycelium seed phrase and create the same wallet on MultiBit HD.

We've not created a Wizard for this as we find users (and ourselves !) get confused by the different sorts of HD wallets. We don't want people thinking that a 'Trezor soft wallet' gives the same level of protection as an actual Trezor.

To make it as clear as we can to users we have a section in the Manage Wallet | Wallet dashboard titled 'Wallet capabilities'. This gives a description of the wallet e.g. for standard MultiBit HD wallets it says:

```
Standard MultiBit HD wallet
Private keys are encrypted on this machine
BIP32 compliant. m/0h (Account 1 only)
```

tl;dr A Mycelium wallet can be recreated on MultiBit HD if the user really wants to.

**Comment by crwatkins** [comment]
Thanks Jim.  Just to be certain, however:  A wallet created on MultiBit HD could not be recovered with Mycelium, right?

**Comment by gurnec** [comment]
@crwatkins Apologies for interjecting, but to answer your question: MulitBit HD always *creates* BIP-32 path wallets, and Mycelium only supports BIP-44 path wallets, so "yes, right".

I should also point out that Mycelium supports multiple BIP-44 accounts (I haven't tested this with a Trezor though), and a BIP-44 wallet imported into MultiBit HD will only see the first account with account index 0, called "Account 1" by Mycelium. Any other accounts are ignored.

If it's of any future use, I try to maintain a [spreadsheet here](https://onedrive.live.com/redir?resid=584F122BA17116EE%21313) listing deterministic wallet compatibility details, along with compatibility charts between different wallets (and a discussion thread is [here on BitcoinTalk](https://bitcointalk.org/index.php?topic=1000544.0)).

**Comment by saivann** [comment]
If I understand correctly, MultiBit HD is compatible with Mycelium, but not the other way around. Accordingly, I have dropped MultiBit HD from the description.

@crwatkins @jim618 @gurnec Thanks a lot for the comments.

@gurnec Cool research! Would be great I think if it was published somewhere like the wiki.

**Comment by harding** [comment]
Updated text LGTM.  Thanks everyone!

**Comment by saivann** [comment]
In the absence of critical feedback, this pull request will be merged on June 18th.

**Comment by crwatkins** [comment]
Thanks @gurnec 

@saivann LGTM
